README.md

```diff
@@ -44,7 +135,7 @@ find their sources in this distribution, in a directory called
 programs are already included in this distribution, in a folder
 called 'binaries'. LOMAP also includes some code that is adapted
 and/or taken from NetworkX Python package v1.6, available at
-http://networkx.github.io. See below for copyright notices and 
+http://networkx.github.io. See below for copyright notices and
 licenses of LTL2BA, scheck, and NetworkX.
```

Comment:
Move the _lomap_ description paragraph  (_"LTL Optimal [...] for examples"_) below the title heading.
Delete the paragraph about third party programs (_"LOMAP uses [...] NetworkX"_).